packages/uauth/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@web3-onboard/uauth",
+  "version": "2.0.0-alpha.1",
+  "description": "UAuth SDK module for connecting to Web3-Onboard. Web3-Onboard makes it simple to connect Ethereum hardware and software wallets to your dapp. Features standardised spec compliant web3 providers for all supported wallets, framework agnostic modern javascript UI with code splitting, CSS customization, multi-chain and multi-account support, reactive wallet state subscriptions and real-time transaction state change notifications.",
```

Comment:
May be good to add "Unstoppable Domains" somewhere in the description for search optimization